media-api/app/lib/elasticsearch/filters.scala

```diff
@@ -36,6 +36,9 @@ object filters {
   def term(field: String, term: String): FilterBuilder =
     termFilter(field, term)
 
+  def term(field: String, value: Boolean): FilterBuilder =
```

Comment:
That looks like nasty Java laziness.
Perhaps just stick to what we know the results of (If we do pass a case class - what happens?)